src/components/Attendant.test.jsx

```diff
@@ -0,0 +1,32 @@
+import Attendant from "./Attendant";
+import { render, screen } from "@testing-library/react";
+
+describe("<Attendant />", () => {
+  let props;
+
+  beforeEach(() => {
+    props = {
+      attendant: {
+        name: "John",
+        lastName: "Doe",
+        jobTitle: "Engineer",
+        age: 40,
+      },
+    };
+  });
+
+  it("renders correctly on successful initial load", async () => {
+    const { container } = render(<Attendant {...props} />);
+
+    expect(container).toMatchSnapshot();
+  });
+
+  it("renders component with given props", async () => {
```

Comment:
Since this component only renders data from the props, a snapshot test is enough